piezo_web_app/PiezoWebApp/tests/services/spark_job/spark_ui_service_test.py

```diff
@@ -0,0 +1,88 @@
+import kubernetes
+import mock
+import pytest
+
+from PiezoWebApp.src.services.spark_job.spark_ui_service import SparkUiService
+from PiezoWebApp.src.utils.configurations import Configuration
+
+
+class TestSparkUiService:
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        mock_configuration = mock.create_autospec(Configuration)
+        mock_configuration.k8s_url = 'http://0.0.0.0:0'
```

Comment:
The case where this is `https` seems to be completely untested (but see previous comment about the http/https distinction)